client/src/App.tsx

```diff
@@ -11,6 +12,7 @@ function App() {
      <Route path="/" element={<GetAllTasks />} />
       <Route path="/create-task" element={<CreateTask />} />
       <Route path="/tasks" element={<CreateTask />} />
+      <Route path="/for-professionals" element={<TaskListForProfessionals />} />
```

Comment:
Normally the API would have common roots and different branches representing the routes that can be taken.
For example:
GET /tasks => to get all tasks, and you can pass query parameters to do filtration and/or pagination.
POST /tasks => to create a task

Reply:
yes, totally agree with common root on the API. But here is the frontend, and I just doing a request always to the same endpoint GET /task but the params that I need here are not implemented yet.